test/unit_test/character_sheet_unit_tests/FocusesSetUnitTests.java

```diff
@@ -0,0 +1,86 @@
+package unit_test.character_sheet_unit_tests;
+
+import static org.junit.Assert.*;
+
+import java.security.InvalidParameterException;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import rpg_database.character_sheet.CharacterSheet;
+import rpg_database.character_sheet.Fields;
+import rpg_database.character_sheet.Focuses;
+import rpg_database.character_sheet.Focus;
+import rpg_database.character_sheet.FocusesSet;
+
+public class FocusesSetUnitTests {
+
+	@Rule
+	public ExpectedException thrown = ExpectedException.none();
+
+	@Test
+	public void expectException_SetFocusesMalformedInput() {
+		expectExceptionWithMessage(InvalidParameterException.class,
+				"class java.lang.String value is not an instance of class rpg_database.character_sheet.Focuses");
+		final String malformedInput = "MALFORMED INPUT";
+		CharacterSheet characterSheet = new CharacterSheet("CharacterSheet");
+		characterSheet.setData(Fields.FOCUSES, malformedInput);
+	}
+

```

Comment:
Itt kellene azt is nézni, hogy a FocusesSet dob -e exception-t ha pl. ilyet csinálsz, hogy ```new FocusesSet(Focueses.ANIMAL_HANDLING, Focueses.ANIMAL_HANDLING, Focueses.ANIMAL_HANDLING);```

Vagy ha olyat hogy 
```java
FocusesSet focusesSet = new FocusesSet(Focueses.ANIMAL_HANDLING, Focueses.ANIMAL_HANDLING);
focusesSet.add(Focueses.ANIMAL_HANDLING);
```
vagy erre:
```java
FocusesSet focusesSet = new FocusesSet(Focueses.ANIMAL_HANDLING, Focueses.ANIMAL_HANDLING);
focusesSet.addAll(new FocusesSet(Focueses.ANIMAL_HANDLING));
```

mindhárom esetet kivételkezelni kell.